src/ralph/domains/models/domains.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
```

Comment:
since there is only one file with only few models you could use `models.py` directly instead of creating models directory. We were using models directory iff there were lot of models in single app.